plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraTableHandle.java

```diff
@@ -105,6 +108,21 @@ public boolean equals(Object obj)
     @Override
     public String toString()
     {
-        return schemaName + ":" + tableName;
+        String string = format("%s:%s", schemaName, tableName);
```

Comment:
nit: `string` is a strange name for a variable, could this be changed? (I see it elsewhere in the Cassandra Plugin code, but only 2 places).
Most code uses `toStringHelper(this)` or else:
```
StringBuilder builder = new StringBuilder(format("%s:%s", schemaName, tableName));
...
if (this.partitions.isPresent()) {
    builder.append(format(" %d partitions %s", partitions.size(), Stream.concat(....

```
?

Reply:
`HiveTableHandle` handles it like this:
```
    @Override
    public String toString()
    {
        StringBuilder builder = new StringBuilder();
        builder.append(schemaName).append(":").append(tableName);
        bucketHandle.ifPresent(bucket -> {
            builder.append(" buckets=").append(bucket.getReadBucketCount());
            if (!bucket.getSortedBy().isEmpty()) {
```

Reply:
@leveyja if i were using `StringBuilder` i would indeed use `builder` and i had this this way originally.
However, there is no benefit of using `StringBuilder` in this code (no performance gain), and `append` vs `+=` is just more verbose, so i switch to `String`.

Reply:
@findepi `String string` is just something I've never seen before. `StringBuilder builder` - it is a builder, it's common.
If you want to stick with `String` + concatenation, I'd use `String toString` to give it more meaning.
`Object object`, `List list`, all of these are redundant / unseen in my experience.
Re: performance gain - it's not in an inner loop, but there are intermediate strings generated by all the concatenations (I count at least 10), so while I wouldn't classify it as "performance gain", I'd see it as much better to use a builder for arbitrary String concatenation, unless it is *absolutely* trivial (e.g., `throw new IllegalStateException("blah " + input)`

```
StringBuilder builder = new StringBuilder(schemaName).append(":").append(tableName);

```
^ This removes the `String.format("%s:%s")` - so I'd say we go with it. `String string` honestly made me comment - it's *very* weird.

Reply:
`String toString` would be fine, i don't see much difference. Perhaps it could also be `result`
 `StringBuilder builder` would be OK too. I just find it hard to read and will prefer ordinary string concatenation -- as implemented in this PR, or here https://github.com/trinodb/trino/blob/bf798408e45aaa46bfd21e588247bf835b84d94c/plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultQueryBuilder.java#L79-L93

anyway, i am open for this to be changed